test/useTbTable.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { render, fireEvent, getByText, waitFor, getByRole, getAllByRole } from '@testing-library/react';
+import { TbTableComponent } from './components/tbTableComponent';
+
+const getRowBoundaries = (rows: any[]) => {
+    const lastRowCells = getAllByRole(rows[9], 'cell');
+    const firstRowCells = getAllByRole(rows[0], 'cell');
+
+    return {
+        first: firstRowCells,
+        last: lastRowCells,
+    };
+};
+
+describe('TbTableComponent', ()=> {
+    it('should render initial state w/o problem', async () => {
+        const { getByRole } = render(<TbTableComponent />);
+        const table = getByRole('table');
+        expect(table).toBeDefined();
+    });
+
+    it('Should change setMultiSort value when pressing keyDown and keyUp', async () => {
+        const { container } = render(<TbTableComponent />);
+
+        fireEvent.keyDown(container, { key: 'Control', code: 'Control' });
+        fireEvent.keyUp(container, { key: 'Control', code: 'Control' });
+
+        expect(container).toBeDefined();
+    });
+
+    it('Pressing a key different to Control', async () => {
```

Comment:
Please use a better description for this test. "Pressing a key different to Control" doesn't really express what you're testing.